package.json

```diff
@@ -21,23 +21,11 @@
   },
   "homepage": "https://github.com/takenet/blip-components#readme",
   "devDependencies": {
-    "cz-customizable": "^5.3.0",
```

Comment:
why did u change all those things?

Reply:
Those things should not be changed, was a configuration inside my local eviroment. I'm gonna to discart those changes.